### ISSUE TITLE
ReflectionException Class custom does not exist

### ISSUE THREAD

**Issue by roelvanhintum** [fruitstudios/craft-linkit#17]
I'm guessing this has something to do with existing data from Craft 2. Is there any way to migrate old data to the Craft 3 plugin?

```
in /var/www/vendor/fruitstudios/linkit/src/fields/LinkitField.php at line 105 -
fruitstudios\linkit\fields\LinkitField::_getLinkTypeModelByType('custom')
```

**Comment by samhibberd** [comment]
@roelvanhintum, the migration scripts are just going through testing and should be ready by the weekend. Thanks

**Comment by roelvanhintum** [comment]
@samhibberd Thanks!

**Comment by samhibberd** [comment]
@roelvanhintum not quite ready yet, should be wrapped and ready for release tomorrow. Sorry for  the delay here!

**Comment by roelvanhintum** [comment]
@samhibberd No worries, i was surprised by the quick responses.

**Comment by samhibberd** [comment]
@roelvanhintum the migration scripts are ready and will be released as a part of 1.0.7 shortly. Any problems just shout.
